src/components/ProfileModal/ChangePasswordModal.tsx

```diff
@@ -0,0 +1,218 @@
+import { Button } from "@/components/ui/button";
+import { DialogFooter } from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+import { z } from "zod";
+import cn from "classnames";
+
+import React from "react";
+import { useState, useEffect } from "react";
+
+type ChangePWProps = {
+  setProfileState: React.Dispatch<
+    React.SetStateAction<"view" | "edit" | "changePw">
```

Comment:
Consider defining "view" | "edit" | "changePw" in `ProfileModal` and import it to use here. This ensures that we only have to change the type of profileState in one place instead of having to change it in multiple places